src/Analysis/Problem/Bukkit/PluginDependenciesProblem.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace Aternos\Codex\Minecraft\Analysis\Problem\Bukkit;
+
+use Aternos\Codex\Minecraft\Analysis\Solution\Bukkit\PluginInstallSolution;
+use Aternos\Codex\Minecraft\Analysis\Solution\File\FileDeleteSolution;
+use Aternos\Codex\Minecraft\Translator\Translator;
+
+/**
+ * Class PluginDependencyProblem
+ *
+ * @package Aternos\Codex\Minecraft\Analysis\Problem\Bukkit
+ */
+class PluginDependenciesProblem extends BukkitProblem
+{
+    /**
+     * @var string
+     */
+    protected $pluginPath;
+
+    /**
+     * @var string
+     */
+    protected $pluginName;
+
+    /**
+     * @var string[]
+     */
+    protected $dependencyPluginNames;
+
+    /**
+     * @return string
+     */
+    public function getPluginPath(): string
+    {
+        return $this->pluginPath;
+    }
+
+    /**
+     * @return string
+     */
+    public function getPluginName()
+    {
+        return $this->pluginName;
+    }
+
+    /**
+     * get a list of missing dependencies
+     * @return string[]
+     */
+    public function getDependencyPlugins(): array
+    {
+        return $this->dependencyPluginNames;
+    }
+
+    /**
+     * @return string
+     */
+    public function getDependencyPluginNames(): string
+    {
+        $result = [];
+        foreach ($this->dependencyPluginNames as $name) {
+            $result[] = "'". $name ."'";
+        }
+        return implode(", ", $result);
+    }
+
+    /**
+     * Get a human readable message
+     *
+     * @return string
+     */
+    public function getMessage(): string
+    {
+        return Translator::getInstance()->getTranslation("plugin-dependencies-problem", [
```

Comment:
When only a single plugin is listed the language variable for the single dependency should be used instead.